src/components/ThemeProvider/utils/themeToVariables.ts

```diff
@@ -18,6 +18,14 @@
 
 import { isObject } from 'lodash-es'
 
+const varsInPixelSegments = ['shape', 'spacing', 'fontSize', 'lineHeight']
+
+function addUnitToValue(value: unknown, fieldName: string): unknown {
+  return varsInPixelSegments.some((segment) => fieldName.includes(segment))
```

Comment:
Why includes here instead of a stricter comparison?

Reply:
Since the names of the variables are not all the same (e.g., all starting with one of these keywords) it seemed more flexible to just check with an `includes`. 

However, if needed, I can make an equality check listing all the variables of interest

Reply:
No actual need for it something stricter, not that it comes to my mind at least. If you think it's safer this way it's fine to me

Reply:
it is important that designers generate the theme without px or be aligned on this change

Reply:
Af of today, we have no control over the plugin we use to export the primitives, and it automatically generates the values with "px" attached.
<img width="281" alt="Screenshot 2024-06-04 at 12 10 55" src="https://github.com/mia-platform/design-system/assets/29396290/09547518-8ff0-43c7-b169-dfeb29505e2b">

Reply:
The change made to the library only regards the CSS variables, not the theme itself. It was my understanding that Ant needs the values to be unitless, so we cannot change that part of the generation.
Therefore, the changes should not affect the design part, or the theme generation